src/journey.js

```diff
@@ -16,23 +16,23 @@ exports.GET = (event, context, callback) => {
     }
 }
 
-exports.POST = (event, context, callback) => {
+exports.create = (event, context, callback) => {
     let reqBody = JSON.parse(event.body);
     let newItem = {
         id: uuidv4(),
         createdAt: new Date().toISOString(),
         label: reqBody.label,
         doc: reqBody.doc,
         parent: reqBody.parent,
-        type: "journey"
+        type: journey
```

Comment:
Does this - 'journey' without quotes - work?

Reply:
No. That's a mistake. Corrected in 10d496a